src/libraries/Common/src/System/HexConverter.cs

```diff
@@ -149,5 +149,54 @@ public static char ToCharLower(int value)
 
             return (char)value;
         }
+
+        public static bool TryDecodeFromUtf16(ReadOnlySpan<char> chars, Span<byte> bytes)
+        {
```

Comment:
This method has some strict requirements on the input, namely it expects that bytes is exactly chars.Length / 2 and chars.Length is divisible by 2.  Both should be Debug.Assert'd.

Reply:
Done